docs_user/modules/openstack-backend_services_deployment.adoc

```diff
@@ -147,6 +154,12 @@ metadata:
 spec:
   secret: osp-secret
   storageClass: local-storage
+  tls:
+    endpoint:
+      internal:
+        enabled: true
```

Comment:
internal changed to `podLevel`  https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/core_v1beta1_openstackcontrolplane_network_isolation_tlse.yaml#L10:

```
  tls:
    podLevel:
      enabled: true
```

Reply:
I see in tests/config/base/openstack_control_plane.yaml it is correct